controller/controller.mustache

```diff
@@ -3,14 +3,19 @@ const r = require('../lib/db');
 const controller = require('../lib/controller');
 const rethinkdb = require('../lib/rethinkdb');
 const schema = require('../lib/schema');
+const httpError = require('standard-http-error');
+{{#isUser}}
+const jwt = require('../lib/jwt');
+const password = require('password');
+{{/isUser}}
 
 const appSchema = require('../schema').getSchema();
 const modelSchema = appSchema['{{name}}'];
 
-const LINK = 'https://rethinkdb.com/docs/changefeeds/javascript/'
-var RESTRICTED_FEED = "WARNING: Changefeed is restricted and will only"
-RESTRICTED_FEED += " include updates to existing documents, not newly created"
-RESTRICTED_FEED += " documents. Read more here: " + LINK;
+const LINK = 'https://rethinkdb.com/docs/changefeeds/javascript/';
+var RESTRICTED_FEED = 'WARNING: Changefeed is restricted and will only';
+RESTRICTED_FEED += ' include updates to existing documents, not newly created';
+RESTRICTED_FEED += ' documents. Read more here: ' + LINK;
```

Comment:
Do we have many 'warnings' ?  If we have more than one, I think we could move these to a `/constants` folder or similar, so as not to clog up space.